### PR TITLE
Delete pipelines

### PR DESCRIPTION
**Pull request by bariscaglar**

### CODE REVIEW

File: .pipelines/iis.compression.build.dev.yml [hunk 1]
```diff
@@ -1,16 +1,5 @@
 name: $(BuildDefinitionName)_$(SourceBranchName)_$(Date:yyyyMMdd)$(Rev:.rr)
 
-trigger:
```
Comment (bangbingsyb) [inline .pipelines/iis.compression.build.dev.yml]: Looks like the PR removes both CI and PR triggers in the pipeline yml. Do we add them somewhere else?

Comment (bariscaglar) [reply .pipelines/iis.compression.build.dev.yml]: Sorry, this is me trying to get the PR trigger working.

Comment (bariscaglar) [reply .pipelines/iis.compression.build.dev.yml]: Which I couldn't, by the way.

**Comment by shirhatti** [comment]
Closing as this PR is stale. #46 should obsolete it.

**Pull request closed without merging**